src/components/pages/Home/Home.style.scss

```diff
@@ -37,22 +54,44 @@
         font-size: 20px;
         line-height: 29px;
         color: $gainsboro;
+        @media screen and (max-width: 768px) {
+          margin-bottom:1em;
+          padding: 0px 10px;
+          text-align: center;
+          font-size: 18px;
+          line-height: 25px;
+        }
       }
 
       &__btn-token {
         width: 160px;
         margin-right: 21px;
+        @media screen and (max-width: 800px) {
+          margin:0em 0em 1em;
+        }
       }
 
       &__btn-started {
         width: 160px;
+        margin-bottom:1em;
         background-color: $white;
         color: $cornflower-blue;
       }
+
+      &__logo-frost-only {
+        padding-right:1em;
```

Comment:
This `padding-right` can be removed now because the box around it has padding on the right side now as well.